queries/http/highlights.scm

```diff
@@ -0,0 +1,33 @@
+; Display errors
+(ERROR) @error
+
+; Comments
+(comment) @comment
+
+(request
+  method: (method) @keyword
+  url: (url) @TSURI)
```

Comment:
we have `@text.uri` that gets mapped to `TSURI`

Reply:
Changed. Also changed the same on line 33. Should I have or did you omit that on purpose?

Reply:
no, was fine how you did it!